test/bootstrap/functional.php

```diff
@@ -8,6 +8,10 @@
  * file that was distributed with this source code.
  */
 
+if (!defined('SF_TEST_WITHOUT_COMPOSER')) {
```

Comment:
@thePanz and @connorhu this will bring support for composer autoloading in the tests and a way towards getting rid of the submodules.